holoplot/__init__.py

```diff
@@ -0,0 +1,385 @@
+import holoviews
+from holoviews import (
+    extension as _extension, Dimensioned as _Dimensioned, HoloMap as _HoloMap
+)
+
```

Comment:
I also don't much like parentheses in import statements, but again you must disagree!

Reply:
I do disagree, in this case it doesn't seem all that necessary but as soon as it doesn't fit on one line I prefer it.

Reply:
Ah.  I prefer just having multiple imports in that case, with each fitting onto one line, because it's visually easier for me to scan and dismiss a large block of imports that way; when there are parentheses it starts looking like actual code. But as you wish...